back/services/arangoManager.py

```diff
@@ -33,6 +36,11 @@ def setup():
     print(" - Connection established")
 
 
+def setup_collection(collection_name):
```

Comment:
What do you think about `setup_collection_name` ? Without that, we may think that this function changes something in arrango.

Reply:
Should I change it for setup_works_collection_name since we will have more than one collection in the future ?